netket/utils/group/_group.py

```diff
@@ -104,8 +104,15 @@ def remove_duplicates(self, *, return_inverse=False) -> "FiniteGroup":
     def inverse(self) -> Array:
         r"""
         Indices of the inverse of each element.
-        If :code:`g = self[idx_g]` and :code:`h = self[self.inverse[idx_g]]`, then
-        :code:`gh = product(g, h)` is equivalent to :code:`Identity()`
+
+        Assuming the definitions
+
+        .. code::
+
+            g = self[idx_g]
+            h = self[self.inverse[idx_g]]
+
+        Then :code:`gh = product(g, h)` is equivalent to :code:`Identity()`
```

Comment:
```suggestion
        :code:`gh = product(g, h)` is equivalent to :code:`Identity()`
```

Reply:
why? Maybe I was missing a comma but it should read a bit like

Assuming the definitions .... then, A is equivalent to B.

Reply:
hmm. ok, grammatically you are right..